tweak a couple commands in VS code

alexboche opened the pull request.

alexboche commented:
This is working on my personal branch, but when I tried to tested on a branch equivalent to the latest dictation tools/caster/develop everything worked except for a a couple commands such as 'settings'.I think everything should be okay but would you mind testing this because I think there was something weird going on on my computer?

kendonB commented on castervoice/apps/vscode.py:
```diff
@@ -300,7 +304,7 @@ class VSCodeNonCcrRule(MergeRule):
               rdescript="Shift Current Group of Tabs to the Left E.g. Swap with Pane to the Left"),
-        "shift group left":
+        "shift group right":
             R(Key("c-k, right"),
```
Can you change the rdescript styles to `VS Code: My description`?

LexiconCode replied:
With the recent changes rdescript is no longer necessary unless the command does not reflect the action it produces.

LexiconCode commented:
Take a moment to resolve the conflicts and will see about getting this merged.

alexboche commented:
> Take a moment to resolve the conflicts and will see about getting this merged.

It looks like the merge conflicts are basically me adding rdescripts. In a few cases those are unnecessary, but in a few of the merge conflicts I think they are important since the command does not make it obvious what the command does and how it differs from other similar commands. When I resolve the merge conflict, do I need to side with dictation toolbox/caster or can I merge in favor of my changes? In VS code I believe I have the option to accept either my changes or the dictation toolbox/caster changes

LexiconCode commented:
> 
> 
> > Take a moment to resolve the conflicts and will see about getting this merged.
> 
> It looks like the merge conflicts are basically me adding rdescripts. In a few cases those are unnecessary, but in a few of the merge conflicts I think they are important since the command does not make it obvious what the command does and how it differs from other similar commands. When I resolve the merge conflict, do I need to side with dictation toolbox/caster or can I merge in favor of my changes? In VS code I believe I have the option to accept either my changes or the dictation toolbox/caster changes

Merge conflicts exist simply because you your pull request was issued on a version that has had changes. The idea is to respect changes made since by others while integrating your own. In terms of rdescripts feel free adapt that as needed to explain commands in a concise few words or to remove them altogether the command is self-explanatory.

alexboche commented:
I think this is done,I would remove the "waiting for user reply" tag

mrob95 commented:
This doesn't work. The non-CCR rule needs to be created first and then referenced in the `non` property of the CCR rule.

alexboche commented:
Sorry about that. I just fixed the problem as Mike directed. I also added one command "comment" and moved another "copy path". I tested some command from both rules and they work.

LexiconCode commented:
Mike's requested we hold off on changes to applications as he's working in a re-factor. Once that's done this will merge.

LexiconCode commented:
All right @alexboche https://github.com/dictation-toolbox/Caster/pull/604 has been merged.

alexboche commented:
Okay I think this one is done now.
